src/main/java/com/khomishchak/ws/services/integration/whitebit/WhiteBitServiceImpl.java

```diff
@@ -221,6 +157,20 @@ private DecryptedApiKeySettingDTO getApiKeysPair(long userId) {
                         () -> new IllegalArgumentException(String.format("API Keys with for code: %s not present", CODE)));
     }
 
+    private <T> Mono<T> getErrorResp(WhiteBitV4ErrorResp resp, int statusCode) {
+        List<String> errors = resp.getErrors() != null ? resp.getErrors().getMessages() : new ArrayList<>();
```

Comment:
```suggestion
        Object errors = resp.getErrors();
        List<String> errorMessages = errors != null ? errors.getMessages() : new ArrayList<>();
```
Also it can be done by `Optional` but it's on  your taste